Source/WTF/wtf/URL.h

```diff
@@ -234,7 +234,6 @@ class URL {
 
 #if USE(FOUNDATION)
     WTF_EXPORT_PRIVATE URL(NSURL *);
-    WTF_EXPORT_PRIVATE operator NSURL *() const;
```

Comment:
Normally I suggest removing all the clients in a separate pull request. That way if we have to revert because of Apple internal use or something like that we might be able to just revert the operator removal and keep the rest. But this might be successful as is.